### PR TITLE
Fix | Adjust path for .AssemblyAttributes in obj folder

### PR DESCRIPTION
**Pull request by JRahnama**
fixes issue #2535

### CODE REVIEW

File: src/Microsoft.Data.SqlClient/add-ons/Directory.Build.props [hunk 1]
```diff
@@ -11,7 +11,7 @@
     <ReferenceType Condition="'$(ReferenceType)'==''">Project</ReferenceType>
     <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
     <ContinuousIntegrationBuild>true</ContinuousIntegrationBuild>
-    <TargetFrameworkMonikerAssemblyAttributesPath>$([System.IO.Path]::Combine('$(IntermediateOutputPath)','$(TargetFrameworkMoniker).AssemblyAttributes$(DefaultLanguageSourceExtension)'))</TargetFrameworkMonikerAssemblyAttributesPath>
+    <TargetFrameworkMonikerAssemblyAttributesPath>$([System.IO.Path]::Combine('$(IntermediateOutputPath)/$(TargetFramework)','$(TargetFrameworkMoniker).AssemblyAttributes$(DefaultLanguageSourceExtension)'))</TargetFrameworkMonikerAssemblyAttributesPath>
```
Comment (campersau) [inline src/Microsoft.Data.SqlClient/add-ons/Directory.Build.props:14]: Don't hardcode directory separators?

```suggestion
    <TargetFrameworkMonikerAssemblyAttributesPath>$([System.IO.Path]::Combine('$(IntermediateOutputPath)','$(TargetFramework)','$(TargetFrameworkMoniker).AssemblyAttributes$(DefaultLanguageSourceExtension)'))</TargetFrameworkMonikerAssemblyAttributesPath>
```

File: src/Microsoft.Data.SqlClient/netcore/src/Microsoft.Data.SqlClient.csproj [hunk 2]
```diff
@@ -20,7 +20,7 @@
     <NoWarn>$(NoWarn);IL2026;IL2057;IL2072;IL2075</NoWarn>
   </PropertyGroup>
   <PropertyGroup>
-    <TargetFrameworkMonikerAssemblyAttributesPath>$([System.IO.Path]::Combine('$(IntermediateOutputPath)','$(TargetFrameworkMoniker).AssemblyAttributes$(DefaultLanguageSourceExtension)'))</TargetFrameworkMonikerAssemblyAttributesPath>
+    <TargetFrameworkMonikerAssemblyAttributesPath>$([System.IO.Path]::Combine('$(IntermediateOutputPath)/$(TargetFramework)','$(TargetFrameworkMoniker).AssemblyAttributes$(DefaultLanguageSourceExtension)'))</TargetFrameworkMonikerAssemblyAttributesPath>
```
Comment (campersau) [inline src/Microsoft.Data.SqlClient/netcore/src/Microsoft.Data.SqlClient.csproj:23]:
```suggestion
    <TargetFrameworkMonikerAssemblyAttributesPath>$([System.IO.Path]::Combine('$(IntermediateOutputPath)','$(TargetFramework)','$(TargetFrameworkMoniker).AssemblyAttributes$(DefaultLanguageSourceExtension)'))</TargetFrameworkMonikerAssemblyAttributesPath>
```

**Review by David-Engel — APPROVED** [review]

**Comment by ErikEJ** [comment]
Just curious, was this because if the .netstandard removal?

**Comment by JRahnama** [comment]
> Just curious, was this because of the .netstandard removal?

No, seems like an oversight when adding Source Link

**Review by DavoudEshtehari — APPROVED** [review]

**Pull request merged**